wrappers/node/src/rustlib.ts

```diff
@@ -58,6 +58,13 @@ export type rust_listener_handle = rust_object_handle;
 export type rust_connection_handle = rust_object_handle;
 
 export interface IFFIEntryPoint {
+  vcx_open_main_pool: (commandId: number, config: string, cb: any) => number,
+
+  vcx_create_agency_client_for_main_wallet: (commandId: number, config: string, cb: any) => number,
+  vcx_provision_cloud_agent: (commandId: number, config: string, cb: any) => number,
```

Comment:
Seems like there no C function like this defined in rust?

Reply:
It is, see libvcx/src/api/utils.rs:139